Code/Engine/Foundation/Containers/ArrayBase.h

```diff
@@ -31,18 +31,16 @@ class ezArrayBase
 
   /// \brief Compares this array to another contiguous array type.
   bool operator==(const ezArrayBase<T, Derived>& rhs) const; // [tested]
-
-  /// \brief Compares this array to another contiguous array type.
-  bool operator!=(const ezArrayBase<T, Derived>& rhs) const; // [tested]
+  EZ_ADD_DEFAULT_OPERATOR_NOTEQUAL(const ezArrayBase<T, Derived>&);
 
   /// \brief Compares this array to another contiguous array type.
   bool operator<(const ezArrayBase<T, Derived>& rhs) const; // [tested]
 
+#if EZ_DISABLED(EZ_USE_CPP20_OPERATORS)
```

Comment:
In C++ 20 having all these overloads actually creates ambiguous situations. In theory C++ 20 is much cleaner in this regard. Unfortunately for C++ 17 it breaks without these overloads.

However, the C++ 20 situation still requires more investigation. Stuff's complicated.